docs/_integration-with-existing-apps-ios.md

```diff
@@ -0,0 +1,596 @@
+
+## Prerequisites
+
+Follow the guide on [setting up your development environment](set-up-your-environment) and using [React Native without a framework](getting-started-without-a-framework) to configure your development environment for building React Native apps for iOS.
+This guide also assumes you're familiar with the basics of iOS development such as creating a `UIViewController` and editing the `Podfile` file.
+
+### 1. Set up directory structure
+
+To ensure a smooth experience, create a new folder for your integrated React Native project, then **move your existing iOS project** to the `/ios` subfolder.
+
+## 2. Install NPM dependencies
+
+Go to the root directory and run the following command:
+
+```
+wget https://raw.githubusercontent.com/react-native-community/template/refs/heads/0.76-stable/template/package.json
+```
+
+This will copy the `package.json` [file from the Community template](https://github.com/react-native-community/template/blob/0.76-stable/template/package.json) to your project.
+
+Next, install the NPM packages by running:
+
+<Tabs groupId="package-manager" queryString defaultValue={constants.defaultPackageManager} values={constants.packageManagers}>
+<TabItem value="npm">
+
+```shell
+npm install
+```
+
+</TabItem>
+<TabItem value="yarn">
+
+```shell
+yarn install
+```
+
+</TabItem>
+</Tabs>
+
+Installation process has created a new `node_modules` folder. This folder stores all the JavaScript dependencies required to build your project.
+
+Add `node_modules/` to your `.gitignore` file (here the [Community default one](https://github.com/react-native-community/template/blob/0.76-stable/template/_gitignore)).
+
+### 3. Install Development tools
+
+### Command Line Tools for Xcode
+
+Install the Command Line Tools. Choose **Settings... (or Preferences...)** in the Xcode menu. Go to the Locations panel and install the tools by selecting the most recent version in the Command Line Tools dropdown.
+
+![Xcode Command Line Tools](/docs/assets/GettingStartedXcodeCommandLineTools.png)
+
+### CocoaPods
+
+[CocoaPods](https://cocoapods.org) is a package management tool for iOS and macOS development. We use it to add the actual React Native framework code locally into your current project.
+
+We recommend installing CocoaPods using [Homebrew](https://brew.sh/):
+
+```shell
+brew install cocoapods
+```
+
+## 4. Adding React Native to your app
+
+### Configuring CocoaPods
+
+To configure CocoaPods, we need two files:
+
+- A **Gemfile** that defines which Ruby dependencies we need.
+- A **Podfile** that defines how to properly install our dependencies.
+
+For the **Gemfile**, go to the root directory of your project and run this command
+
+```sh
+wget https://raw.githubusercontent.com/react-native-community/template/refs/heads/0.76-stable/template/Gemfile
+```
```

Comment:
This could be marked as optional

Reply:
Yes, but there are som many issues that can occur if you don't use the right gems that I'd rather keep it.